lib/node_modules/@stdlib/math/base/special/cpolar/lib/index.js

```diff
@@ -19,31 +19,41 @@
 'use strict';
 
 /**
-* Compute the absolute value and the phase of a complex number.
+* Compute the absolute value and the phase of a double-precision complex floating-point number.
 *
 * @module @stdlib/math/base/special/cpolar
 *
 * @example
+* var Complex128 = require( '@stdlib/complex/float64' );
 * var cpolar = require( '@stdlib/math/base/special/cpolar' );
 *
-* var o = cpolar( 5.0, 3.0 );
+* var o = cpolar( new Complex128( 5.0, 3.0 ) );
 * // returns [ ~5.83, ~0.5404 ]
 *
 * @example
+* var Complex128 = require( '@stdlib/complex/float64' );
 * var cpolar = require( '@stdlib/math/base/special/cpolar' );
+* var Float64Array = require( '@stdlib/array/float64' );
```

Comment:
```suggestion
* var Complex128 = require( '@stdlib/complex/float64' );
* var Float64Array = require( '@stdlib/array/float64' );
* var cpolar = require( '@stdlib/math/base/special/cpolar' );
```

Reply:
@Planeshifter We should have a lint rule for ordering require statements, such that we ensure that the package require comes last.

Reply:
@steff456 Would you mind creating an internal todo for this?

Reply:
@steff456 @kgryte Not necessary. This will now be caught by `@stdlib/_tools/eslint/rules/jsdoc-main-export` per https://github.com/stdlib-js/stdlib/commit/e8ba2ab6bab4ae89928eaca265755eea4139e3ff. Still need to add test cases and do some clean-up, but this will be enforced going forward.